assets/sass/patterns/organisms/content-header.scss

```diff
@@ -25,6 +24,49 @@
 
 }
 
+// Content Header Grid Articles
+
+.content-header-grid {
+  padding: 0 24px;
+}
+
+.content-header-grid__main {
+  @include margin(24, "bottom");
+  @include padding(24, "bottom");
+  border-bottom: 1px solid $color-text-dividers;
+}
+
+.content-header-grid__side {
+  @include margin(24, "bottom");
+  @include padding(24, "bottom");
+  border-bottom: 1px solid $color-text-dividers;
+}
+
+@media only all and (min-width: 900px) {
```

Comment:
In other locations in this file, these @media definitions are nested under the respective class. Would it not make more sense to replicate that model? E.g..

```
.conter-header-grid {
  padding: 0, 24px;

  @media only all and (min-width: 900px)
    border-bottom: 1px solid $color-text-dividers;
    display: grid;
    grid-template-columns: repeat(12, 1fr);
    grid-gap: 24px;
    max-width: 1114px;
    padding: 0;
  }
```

Reply:
Agree that this how it is currently done but will want to review this at some point in the future if the Journal survives

Reply:
What do you mean by `if the Journal survives`?

Reply:
If it does not get put into maintenance mode.